lib/modules/ai_bot/commands/image_command.rb

```diff
@@ -129,10 +129,7 @@ def process(prompts:, seeds: nil)
       [/grid]
     RAW
 
-      {
-        prompts: uploads.map { |item| { prompt: item[:prompt], seed: item[:seed] } },
-        displayed_to_user: true,
```

Comment:
yes, this is part of the problem, it will start hallucinating it inwards if you show it outwards

it was there to attempt to inform the LLM that images were displayed to users, but it appears that even GPT 3.5 does not need this